### PR TITLE
chore(website): update design getting started page

### PR DESCRIPTION
**Pull request by loreina**
- Updated the page description / tagline
- Updated some copy in Fonts section
- Moved Fonts section to the bottom of the page after review
- Match content to Working with Figma file

### CODE REVIEW

**Review by richbachman — APPROVED** [review]

**Review by andipants12 — APPROVED** [review]

File: packages/paste-website/src/pages/getting-started/design/index.mdx [hunk 1]
```diff
@@ -50,13 +50,13 @@ export const pageQuery = graphql`
...
+## Fonts
+
+Fonts in Paste are preloaded into Figma so you don't have to worry about downloading or installing them.
 
 ### Fira Mono
 
-If you are using <strong>any theme</strong> in Paste, you will need
-to [download and install Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono),
-our monospace typeface for code. It’s a free Google font.
+[Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono) is a free Google font and our monospace typeface for code. If you are using <strong>any theme</strong> in Paste, you will need to use it.
```
Comment (katieporter) [inline packages/paste-website/src/pages/getting-started/design/index.mdx:59]:
```suggestion
[Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono) is a free Google font and our monospace typeface for code.
```

Comment (katieporter) [inline packages/paste-website/src/pages/getting-started/design/index.mdx:59]: Can we do a similar rewrite for all of the typefaces, to remove the language around needing to download and install it?

Comment (serifluous) [reply packages/paste-website/src/pages/getting-started/design/index.mdx:59]: oh nice! Didn't realize Google Fonts has the variable font now.

Comment (serifluous) [reply packages/paste-website/src/pages/getting-started/design/index.mdx:59]: Might sound a little less like people need to do something in order to use the fonts if the sentences are reworded like:

> { font name } is used in { x } theme in Paste.

File: packages/paste-website/src/pages/getting-started/design/index.mdx [hunk 2]
```diff
@@ -50,39 +50,17 @@ export const pageQuery = graphql`
...
-If you are using the <strong>Default theme</strong> in Paste, you will need
-to [download and install Inter](https://github.com/rsms/inter/releases/download/v3.15/Inter-3.15.zip), which we use for UI text.
-It’s a free font. After you download the files, follow the instructions in the
-`install-mac.txt` file in your download. We recommend following the steps using Font Book.
-
-### Whitney
-
-If you are using the <strong>Console theme</strong> in Paste, you will need to download Whitney, which we use for UI text.
-Reach out to us in `#help-design-system` if you need access to the Whitney font files for your work.
+Paste supports Product Designers with tooling and assets that help them build customer UIs.
```
Comment (katieporter) [inline packages/paste-website/src/pages/getting-started/design/index.mdx:53]: This first line doesn't feel totally necessary - seems a little redundant with the line in the header

**Review by katieporter — APPROVED** [review]

**Review by andipants12 — APPROVED** [review]

File: packages/paste-website/src/pages/getting-started/design/index.mdx [hunk 3]
```diff
@@ -112,36 +89,76 @@ The main libraries we recommend you have enabled for each file are:
...
+[Inter](https://fonts.google.com/specimen/Inter) is a free Google font and our typeface for UI text.
+
+### SF Pro Text
+
+[SF Pro Text](https://developer.apple.com/fonts/) is the system font for Apple platforms. We represent native web element styles (like option lists in [Select](/components/select/)) in our design assets in macOS styles.
+
+### Whitney
+  
+If you are using the <strong>Console theme</strong> in Paste, you will need to download Whitney, which we use for UI text.
+Reach out to us in `#help-design-system` if you need access to the Whitney font files for your work.
```
Comment (serifluous) [inline packages/paste-website/src/pages/getting-started/design/index.mdx:158]:
```suggestion
[Whitney](https://www.twilio.com/brand/elements/typography) is the Twilio brand typeface. Reach out to us in `#help-design-system` if you need access to the Whitney font files for work outside of Figma.
```

Comment (serifluous) [reply packages/paste-website/src/pages/getting-started/design/index.mdx:158]: @katieporter ok to remove mention of the console theme here?

Comment (katieporter) [reply packages/paste-website/src/pages/getting-started/design/index.mdx:158]: yes plz!

File: packages/paste-website/src/pages/getting-started/design/index.mdx [hunk 4]
```diff
@@ -50,29 +50,24 @@ export const pageQuery = graphql`
...
-to [download and install Inter](https://github.com/rsms/inter/releases/download/v3.15/Inter-3.15.zip), which we use for UI text.
-It’s a free font. After you download the files, follow the instructions in the
-`install-mac.txt` file in your download. We recommend following the steps using Font Book.
+[Inter](https://fonts.google.com/specimen/Inter) is a free Google font and our typeface for UI text. If you are using the <strong>Default theme</strong> in Paste, you will need to use it.
 
 ### Whitney
-
+  
 If you are using the <strong>Console theme</strong> in Paste, you will need to download Whitney, which we use for UI text.
 Reach out to us in `#help-design-system` if you need access to the Whitney font files for your work.
```
[the remark this answers is not in the document]
Comment (katieporter) [reply packages/paste-website/src/pages/getting-started/design/index.mdx]: I like the update that you made below that calls out that folks can ping us if they need Whitney to do any work outside of Figma (like opening up old Sketch files, which is the primary use case at this point).

**Pull request merged**